resources/Cartographer/Modules/Notes.lua

```diff
@@ -1786,6 +1787,7 @@ function Cartographer_Notes:ShowNote(zone, id, creator)
 		poi:SetWidth(t.width*self.db.profile.iconSize)
 		poi:SetHeight(t.height*self.db.profile.iconSize)
 		poi:SetAlpha(t.alpha)
+		if icon == "QuestionMark" or icon == "quest" then poi:SetFrameLevel(10); end -- WHDB related. Quest End > Normal notes.

```

Comment:
Note overlay (e.g. quest start and end notes > normal notes) requires the icon name to be included here. QuestionMark is a WHDB icon.